src/main/java/org/sonarlint/intellij/ui/SonarLintWalkthroughToolWindow.java

```diff
@@ -0,0 +1,482 @@
+    connectWithYourTeamScrollPane.setPreferredSize(new Dimension(70, 100));
+    var connectWithYourTeamBackButton = new JButton(PREVIOUS);
+    var connectWithYourTeamNextButton = new JButton("Next: Reach out to us");
+
+    var connectWithYourTeamBackButtonPanel = new JPanel(new FlowLayout(FlowLayout.LEFT));
+    connectWithYourTeamBackButtonPanel.add(connectWithYourTeamBackButton);
+
+    var connectWithYourTeamNextButtonPanel = new JPanel(new FlowLayout(FlowLayout.RIGHT));
+    connectWithYourTeamNextButtonPanel.add(connectWithYourTeamNextButton);
+
+    createConnectWithYourTeamLayout(connectWithYourTeamStepLabel, connectWithYourTeamLabel,
+      connectWithYourTeamScrollPane, connectWithYourTeamBackButtonPanel, connectWithYourTeamNextButtonPanel, connectWithYourTeamPage,
+      connectWithYourTeamImageLabel);
+
+    var reachOutToUsStepLabel = new JLabel("Step 4/4", SwingConstants.LEFT);
+    reachOutToUsStepLabel.setFont(new Font(FONT, Font.PLAIN, 14));
+
+    var reachOutToUsLAbel = new JLabel("Reach out to us");
+    reachOutToUsLAbel.setFont(new Font(FONT, Font.BOLD, 16));
+
+    var reachOutToUsDescription = createReachOutToUsPageText(font, project);
+
+    var reachOutToUsPane = new JScrollPane(reachOutToUsDescription);
+    reachOutToUsPane.setBorder(null);
+    reachOutToUsPane.setPreferredSize(new Dimension(70, 100));
+    var reachOutToUsBackButton = new JButton(PREVIOUS);
+
+    var reachOutToUsBackButtonPanel = new JPanel(new FlowLayout(FlowLayout.LEFT));
+    reachOutToUsBackButtonPanel.add(reachOutToUsBackButton);
+
+    createReachOutToUsPageLayout(reachOutToUsStepLabel, project, reachOutToUsLAbel, reachOutToUsPane, reachOutToUsBackButtonPanel,
+      reachOutToUsPage,
+      reachOutToUsImageLabel);
+
+    mainPanel.add(welcomePage, "Page 1");
+    mainPanel.add(learnAsYouCodePage, PAGE_2);
+    mainPanel.add(connectWithYourTeamPage, PAGE_3);
+    mainPanel.add(reachOutToUsPage, "Page 4");
+
+    addButtonActionListeners(welcomePageNextButton, mainPanel, learnAsYouCodePageBackButton, learnAsYouCodePageNextButton,
+      connectWithYourTeamBackButton, connectWithYourTeamNextButton, reachOutToUsBackButton);
+
+    var contentFactory = ContentFactory.getInstance();
+    var content = contentFactory.createContent(mainPanel, "", false);
+    toolWindow.getContentManager().addContent(content);
+  }
+
+  private static void addButtonActionListeners(JButton welcomePageNextButton, JPanel mainPanel, JButton learnAsYouCodePageBackButton,
+    JButton learnAsYouCodePageNextButton, JButton connectWithYourTeamBackButton, JButton connectWithYourTeamNextButton,
+    JButton reachOutToUsBackButton) {
+    welcomePageNextButton.addActionListener(e -> {
+      var cl = (CardLayout) (mainPanel.getLayout());
+      cl.show(mainPanel, PAGE_2);
+    });
+
+    learnAsYouCodePageBackButton.addActionListener(e -> {
+      var cl = (CardLayout) (mainPanel.getLayout());
+      cl.show(mainPanel, "Page 1");
```

Comment:
Why are "Page 2" & 3 constants but not 1 & 4? 🧐
Let's make them all constants for consistency